lib/std/c/haiku.zig

```diff
@@ -400,18 +440,12 @@ pub const MAP = struct {
     pub const FAILED = @intToPtr(*c_void, maxInt(usize));
     pub const SHARED = 0x0001;
     pub const PRIVATE = 0x0002;
-    pub const FIXED = 0x0010;
+    pub const FIXED = 0x0004;
     pub const STACK = 0x0400;
```

Comment:
We don't have some of these flags (and we do have other flags not listed here.) Why not just syncronize the full list with the mmap flags we actually have?

Reply:
the mmap flags that i am aware of are the following:

```
MAP_SHARED
MAP_PRIVATE
MAP_FIXED
MAP_ANONYMOUS
MAP_ANON
MAP_NORESERVE
```

i can work on syncing these and see how it goes. if there is any other flags that should be included, let me know.

i can go through another look over the other constants when i get some more spare time. if there are particular ones to focus on, am open to suggestions.

Reply:
Yes, those are all the ones currently in mman.h.

I would look at the error constants especially.